<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Measurement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image</string>
    </property>
   </widget>
   <!-- 图像显示区域 -->
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Display</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 按钮区域 -->
   <widget class="QPushButton" name="button_undo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_modify">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_screenshot">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_generate_table">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Table</string>
    </property>
   </widget>
   <!-- 日志区域 -->
   <widget class="QTextEdit" name="textEdit_results">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
